src/services/companyOffersService.js

```diff
@@ -35,12 +36,15 @@ export const fetchCompanyOffers = measureTime(TIMED_ACTIONS.COMPANY_OFFERS_FETCH
             return json;
         } catch (error) {
 
-            if (!isErrorRegistered) createEvent(EVENT_TYPES.ERROR(
+            const errorArray = Array.isArray(error) ? error :
+                [{ msg: "Unexpected Error. Please try again later." }];
+
+            if (!isErrorRegistered) createErrorEvent(
```

Comment:
when the function call spans over multiple lines, it's better to have the `{}` around the if body